n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -62,47 +68,37 @@ then
     echo "ERROR: BSP is not setup"
 fi
 
-cp ../quartus.ini .
-
-#import opencl kernel files
-quartus_sh -t scripts/import_opencl_kernel.tcl
-
 #check for bypass/alternative flows
 if [ -n "$OFS_ASP_ENV_ENABLE_ASE" ]; then
     echo "Calling ASE simulation flow compile"
     sh ./scripts/ase-sim-compile.sh
     exit $?
 fi
 
-#add BBBs to quartus pr project
-quartus_sh -t scripts/add_bbb_to_pr_project.tcl "$BSP_FLOW"
-
-cp ../afu_opencl_kernel.qsf .
-
-#get a list of gsys files that are mentioned in qsf files; then generate each of them
-eval "$(grep "QSYS_FILE" afu_flat.qsf | grep -v "^#" > qsys_filelist.txt)"
-
-while read -r line; do
-    f=$(echo "$line" | awk '{print $4}')
-    qsys-generate -syn --quartus-project=ofs_top --rev=afu_opencl_kernel "$f"
-    # adding board.qsys and corresponding .ip parameterization files to opencl_bsp_ip.qsf
-    qsys-archive --quartus-project=ofs_top --rev=afu_opencl_kernel --add-to-project "$f"
-done < qsys_filelist.txt
-
-rm -rf qsys_filelist.txt
-
-qsys-generate -syn --quartus-project=ofs_top --rev=afu_opencl_kernel board.qsys
-# adding board.qsys and corresponding .ip parameterization files to opencl_bsp_ip.qsf
-qsys-archive --quartus-project=ofs_top --rev=afu_opencl_kernel --add-to-project board.qsys
-
-#append kernel_system qsys/ip assignments to all revisions
-rm -f kernel_system_qsf_append.txt
-{ echo
-  grep -A10000 OPENCL_KERNEL_ASSIGNMENTS_START_HERE afu_opencl_kernel.qsf
-  echo
-} >> kernel_system_qsf_append.txt
+RELATIVE_BSP_BUILD_PATH_TO_HERE=`realpath --relative-to=$AFU_BUILD_PWD $BSP_BUILD_PWD`
+RELATIVE_KERNEL_BUILD_PATH_TO_HERE=`realpath --relative-to=$AFU_BUILD_PWD $KERNEL_BUILD_PWD`
+#create new 'afu_flat' revision based on the one used to compile the kernel
```

Comment:
Now we create the afu_flat.qsf file based directly on the qsf file used for the kernel-compilation.